### ISSUE TITLE
[Item] Tiny Abomination in a Jar is not generating Mote of Anger on Seal of Vengeance Holy Damage

### ISSUE THREAD

**Issue by AgeAgainst** [dalaranwow/dalaran-wow#4764]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tiny Abomination in a Jar is not generating Mote of Anger from Seal of Vengeance procs
**Expected behaviour**: Tell us what should happen instead
Seal of Vengeance procs should generate Mote of Anger
**Steps to reproduce the problem**:

1. Use Seal of Vengeance
2. Attack any NPC/Training Dummy
3. Notice Seal of Vengeance procs are not giving a mote of anger

**Include proofs for this behaviour**
https://github.com/TrinityCore/TrinityCore/issues/3463
https://github.com/TrinityCore/TrinityCore/issues/9334
http://www.mmo-champion.com/threads/688878-Tiny-Abomination-in-a-Jar-test-results-w-pic
http://www.wowhead.com/item=50351/tiny-abomination-in-a-jar#comments

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Comment by Rushor** [comment]
some source for this:

> TAiaJ is proc'd by: Judgement, Crusader Strike, Divine Storm, Seal of Corruption, application of Righteous Vengeance, refreshment of Righteous Vengeance, Seal of Vengeance,Seal of Vengeance ticks, Auto Attack, Seal of Righteousness, and Seal of Command (non-cleave portion). Every seal can grant a mote. I didn't list SoL, SoW, or SoJ because they really don't matter here.

**Comment by Rushor** [comment]
Game/Unit: Tiny Abomination in a Jar - Also proc on Righteous Vengeance + Holy Vengeance + Blood Corruption

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4764
* commit: 03e14ba9199c62d46460007b1dbddb83cc16dccf

The issue was fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by ashido96** [comment]
apparently this fix wasn't applied to the heroic version of the trinket according to our ret pally, can you check on that?

**Comment by Sharperos** [comment]
Anyone can post the code of fix, thank so much.
